static/absolute/js/searchPage.js

```diff
@@ -1,8 +1,74 @@
 window.onload = function () {
   initMap();
   showRestaurants();
+  document.getElementById("search-btn").addEventListener("click", async () => {
+    const nameInput = document.getElementById("name-input");
+    const name = nameInput.value;
+
+    const addressInput = document.getElementById("address-input");
+    const address = addressInput.value;
+
+    let lat = null;
+    let long = null;
+
+    if (address === "") {
+      // Use Geolocation
+      try {
+        const {
+          coords: { latitude, longitude },
+        } = await getPosition({ enableHighAccuracy: true });
```

Comment:
hmm.. what is going here? with enableHighAccuracy

Reply:
Is that part of HTML geolocation?

Reply:
Yes, you can pass in an options object to the geolocation api.

Reply:
The `getPosition` method is a wrapper around the HTML geolocation api.